tensorflow/training/docker/2.3/py3/Dockerfile.cpu

```
@@ -35,7 +35,7 @@ ARG ESTIMATOR_URL=https://aws-tensorflow-binaries.s3-us-west-2.amazonaws.com/est
 # the nightly builds. Therefore, while updating the smdebug version, please ensure that the format is not disturbed.
 ARG SMDEBUG_VERSION=1.0.9
 
-RUN apt-get update && apt-get -y upgrade && apt-get install -y --no-install-recommends \
+RUN apt-get update && apt-get -y upgrade --only-upgrade systemd && apt-get install -y --no-install-recommends \
```

nit: Please break this into multiple lines:
```suggestion
RUN apt-get update \
 && apt-get -y upgrade --only-upgrade systemd \
 && apt-get install -y --no-install-recommends \
```

---

Can you add this line on top of the systemd upgrade statement?

```
# TODO: Remove systemd upgrade once it is updated in base image
```